File: trunk/Projects/Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/CategoryManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface ICategoryManager
	'	Inherits IManagerBase(Of Category, System.String)
	'End Interface

	Partial Class CategoryManager
		Inherits ManagerBase(Of Category, System.String)
		Implements ICategoryManager
	End Class
End Namespace

File: Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/CategoryManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface ICategoryManager : IManagerBase<Category, string>
    {
	}
	
	partial class CategoryManager : ManagerBase<Category, string>, ICategoryManager
    {
	}
}